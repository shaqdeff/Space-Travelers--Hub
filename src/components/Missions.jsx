function Missions() {
  return <div>Missions</div>;
}

export default Missions;
