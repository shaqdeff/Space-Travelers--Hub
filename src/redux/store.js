import { configureStore } from '@reduxjs/toolkit';
import missionsReducer from './missions/missionsSlice';
import rocketsReducer from './rockets/rocketSlice';

const store = configureStore({
  reducer: {
    missions: missionsReducer,
    rockets: rocketsReducer,
  },
});

export default store;
