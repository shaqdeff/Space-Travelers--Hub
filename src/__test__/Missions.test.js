import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Missions from '../components/Missions';

it('renders correctly', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <Missions />
    </Provider>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
