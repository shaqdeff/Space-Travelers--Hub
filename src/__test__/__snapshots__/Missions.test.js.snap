// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="container"
    style={
      Object {
        "marginTop": "11.5rem",
        "paddingBottom": "2rem",
      }
    }
  >
    <table
      className="my-5 table table-striped table-bordered table-hover"
      style={
        Object {
          "fontSize": "1.5rem",
        }
      }
    >
      <thead>
        <tr
          className="font-weight-bold"
        >
          <th
            className="p-3"
            style={
              Object {
                "fontSize": "1.6rem",
              }
            }
          >
            Mission
          </th>
          <th
            className="p-3"
            style={
              Object {
                "fontSize": "1.6rem",
              }
            }
          >
            Description
          </th>
          <th
            className="p-3"
            style={
              Object {
                "fontSize": "1.6rem",
              }
            }
          >
            Status
          </th>
          <th />
        </tr>
      </thead>
      <tbody />
    </table>
  </div>
</div>
`;
